internal/model/plugin.go

```diff
@@ -8,16 +8,16 @@ import (
 	mattermostModel "github.com/mattermost/mattermost-server/model"
 )
 
-// Plugin represents a Mattermost plugin in the marketplace.
+// Plugin represents a Mattermost plugin in the Marketplace.
```

Comment:
nit: I've been aiming to capitalize only when in the proper noun context: `Plugin Marketplace`. Should we do that here too if we want to capitalize at all?
```suggestion
// Plugin represents a Mattermost plugin in the Plugin Marketplace.
```

Reply:
Good catch. Will do :+1: